video/out/vo.c

```diff
@@ -507,7 +508,7 @@ static void update_vsync_timing_after_swap(struct vo *vo,
     }
 
     in->num_successive_vsyncs++;
-    if (in->num_successive_vsyncs <= vo->opts->swapchain_depth)
+    if (in->num_successive_vsyncs <= MPMAX(vo->opts->swapchain_depth, DELAY_VSYNC_SAMPLES))
```

Comment:
Maybe just make this `VO_MAX_SWAPCHAIN_DEPTH` instead unless 10 specifically makes some kind of difference on windows?

Reply:
On Windows they are smoothed out over time and 10 samples seems like good value. I don't want to connect it to seemingly unrelated value `VO_MAX_SWAPCHAIN_DEPTH`

Reply:
I rather just make it 10 then. As far as I can tell, this makes no difference on linux.

Reply:
something like now?

Reply:
Yeah.

Reply:
I was just going to suggest the exact opposite of this change.
Why use a "good enough" value when we have a better one we can also take into account?

Reply:
What do you mean? Like make windows presentation work more like linux?

Reply:
As far as I can tell `MPMAX(vo->opts->swapchain_depth, DELAY_VSYNC_SAMPLES)` has zero downsides, I am suggesting to just keep it at that.

Reply:
It's always the same as DELAY_VSYNC_SAMPLES because the later will always be bigger than the former so it's a pointless comparison.

Reply:
I get it now, nvm

Reply:
Is it important that `in->num_successive_vsyncs <= vo->opts->swapchain_depth` ?

Because if the `swapchain_depth` limit changes in the future for some reason then adding that MPMAX back in is an easy thing to miss.

Reply:
Yes, however the current max limit on that option (8) is already stupidly high and not of any real practical use so I don't think we need to worry about that hypothetical.